src/components/KYCWall/BaseKYCWall.js

```diff
@@ -100,7 +100,9 @@ class KYCWall extends React.Component {
             this.setState({shouldShowAddPaymentMenu: false});
             return;
         }
-        this.setState({transferBalanceButton: event.nativeEvent.target});
+        // Use event target as fallback if anchorRef is null for safety
```

Comment:
Oh just a minor change here

```suggestion
        
        // Use event target as fallback if anchorRef is null for safety
```